vaxrank/cli.py

```diff
@@ -204,30 +194,40 @@ def add_vaccine_peptide_args(arg_parser):
         "--vaccine-peptide-length",
         default=25,
         type=int,
-        help="Number of amino acids in the vaccine peptides (default %(default)s)")
+        help="Number of amino acids in the vaccine peptides. (default: %(default)s)")
 
     vaccine_peptide_group.add_argument(
         "--padding-around-mutation",
-        default=0,
+        default=5,
         type=int,
         help=(
             "Number of off-center windows around the mutation to consider "
-            "as vaccine peptides (default %(default)s)"
+            "as vaccine peptides. (default: %(default)s)"
         ))
 
     vaccine_peptide_group.add_argument(
         "--max-vaccine-peptides-per-mutation",
         default=1,
         type=int,
-        help="Number of vaccine peptides to generate for each mutation")
+        help=(
+            "Number of vaccine peptides to generate for each mutation. "
+            "(default: %(default)s)"
+        ))
 
     vaccine_peptide_group.add_argument(
         "--min-epitope-score",
         default=1e-10,
         type=float,
         help=(
             "Ignore predicted MHC ligands whose normalized binding score "
-            "falls below this threshold"))
+            "falls below this threshold. (default: %(default)s)"))
+
+    vaccine_peptide_group.add_argument(
+        "--num-epitopes-per-vaccine-peptide",
```

Comment:
why rename this arg?

Reply:
I think "per peptide" is too vague given that each epitope is itself a peptide and the longer sequence from which the vaccine peptides are drawn are also peptides.